shared/teams/invite-by-email/index.native.js

```diff
@@ -0,0 +1,230 @@
+// @flow
+import * as React from 'react'
+import {Avatar, Box, Button, ClickableBox, Icon, Input, List, Text} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+import {NativeImage} from '../../common-adapters/native-wrappers.native'
```

Comment:
this is exposed if you import common-adapters/index.native, use that instead and add it to the other imports you have above